### PR TITLE
migrated dozer to mapstruct #525

### PR DESCRIPTION
**Pull request by btonokit**
Please review #525.

To match the version with the newly configured Lombok in the `maven-compiler-plugin`, we also added a version property to the originally configured Lombok.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -31,6 +31,34 @@
     <build>
```
Comment (btkatoufm) [inline pom.xml:31]: - [ ] What is the "Add .factorypath to .gitignore" in  [Possible Solution](https://github.com/terasolunaorg/terasoluna-gfw-web-blank/issues/525#issue-1367492349)?

Comment (btonokit) [reply pom.xml:31]: 6ca0dfca663ea25afba9a0a66ff5ca477b59322a
`.factorypath` is the file generated when compiled using MapStruct.
It only contains information such as the version of mapstruct-processor used at the time of compilation, and is unnecessary for the project.
So I added it to `.gitignore`.

**Comment by btonokit** [comment]
5cecac9600cc612e226e608271cb540b44bffd72
Deleted  mapstruct and lombok version properties.
See https://github.com/terasolunaorg/terasoluna-gfw/issues/1183#issuecomment-1249027817

**Comment by btkatoufm** [comment]
relates to [TSF4J5-1485](https://terasolunaorg.atlassian.net/browse/TSF4J5-1485).

**Pull request merged**